pkg/tfbridge/provider_test.go

```diff
@@ -4741,7 +4741,8 @@ func TestUnknowns(t *testing.T) {
 		},
 		"response": {
 			"properties":{
-				"id":""
+				"id":"",
+                                "setBlockProps":[{"prop":""}]
```

Comment:
This was removed in #2060 and is the direct consequence of the objchange.proposedNew edit. Not ideal, but I do not, however, believe the edit is what we want to keep.